paymentsheet/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -31,13 +33,14 @@ import com.stripe.android.paymentsheet.paymentdatacollection.TransformToPaymentM
 import com.stripe.android.paymentsheet.ui.AnimationConstants
 import com.stripe.android.paymentsheet.viewmodels.BaseSheetViewModel
 import com.stripe.android.ui.core.Amount
+import com.stripe.android.ui.core.PaymentsTheme
 import kotlinx.coroutines.launch
 
 internal abstract class BaseAddPaymentMethodFragment : Fragment() {
     abstract val viewModelFactory: ViewModelProvider.Factory
     abstract val sheetViewModel: BaseSheetViewModel<*>
 
-    protected lateinit var addPaymentMethodHeader: TextView
+    protected var addPaymentMethodHeaderVisibility = true
```

Comment:
Tracing through where this is set to somthing other than true.  If this can happy dynamically then the compose content view below will not update

Reply:
We're safe here. This doesn't get changed dynamically. but I did move this to the viewmodel in my next commit because I think it makes more sense there.